test/org.eclipse.elk.alg.mrtree.test/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,297 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.codegen.unusedLocal=preserve
+org.eclipse.jdt.core.compiler.compliance=1.8
```

Comment:
It looks like we did not purge Java 8 from the test plugins. Could you investigate this?